sdk/core/ci.yml

```diff
@@ -4,9 +4,6 @@ resources:
       type: github
```

Comment:
Can this repo reference also be removed or is it still used somewhere else? Our goal is to eliminate these repo references everywhere we can.